cpp/src/io/parquet/experimental/hybrid_scan.cpp

```diff
@@ -58,6 +58,16 @@ std::vector<cudf::size_type> hybrid_scan_reader::all_row_groups(
   return _impl->all_row_groups(options);
 }
 
+size_type hybrid_scan_reader::total_rows_in_row_groups(
+  cudf::host_span<size_type const> row_group_indices) const
+{
+  if (row_group_indices.empty()) { return 0; }
+
+  auto const input_row_group_indices =
+    std::vector<std::vector<size_type>>{{row_group_indices.begin(), row_group_indices.end()}};
+  return _impl->total_rows_in_row_groups(input_row_group_indices);
```

Comment:
Oh this copies data. Why can't we just pass in `row_group_indices`?

Reply:
`_impl->total_rows_in_row_groups` can accept `host_span<host_span<size_type>>`.

Reply:
Oh I see why--the internal function accepts array of array. This is the limitation of using `span` :(

Reply:
It's only temporary, we plan to take in a span of vectors (for multi files case) soon, which will be passed on as is. This is just a glue layer until then.